Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -175,10 +175,17 @@ extension BrowserViewController {
             UIKeyCommand(action: #selector(goForwardKeyCommand), input: UIKeyCommand.inputRightArrow, modifierFlags: .command),
         ]
 
+        let keyboards = [
```

Comment:
nit: I'm a tiny bit iffy about this naming. Maybe we can try windowShortcuts? or something else that's meaningful. Up to you!